src/containers/QuoteDisplayerFn.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'; 
+import Button from '../components/Button/Button.js';
+import Quote from '../components/Quote/Quote.js';
+import { fetchQuote } from '../services/fetchQuote.js';
+
+
+const QuoteDisplayerFn = () => {
+  const [quoteObj, onClick] = useState({
```

Comment:
conventionally, 'we' usually call the function returned by useState: 'set_____'. It makes it much easier when looking around the code to see what is what.

Reply:
good to know this convention!